### DISCUSSION TITLE
I am facing issues using the Helm Repo URL in deploying my helm chart , I am using the same URL as you are using , but I am getting error , can ou please look into this and try to solve this

### DISCUSSION THREAD

**Discussion by pa1narendra** [8gears/n8n-helm-chart#127] [Q&A]
The helm chart I am using is `https://artifacthub.io/packages/helm/open-8gears/n8n`
And the URL that I am using as the helm repo is `8gears.container-registry.com/library`
I also tried with the other URLs like `oci://8gears.container-registry.com/library`
`oci://8gears.container-registry.com/library/n8n`
`8gears.container-registry.com/library/n8n`
`https://github.com/8gears/n8n-helm-chart`
`https://github.com/8gears/n8n-helm-chart/tree/main/charts/n8n`

But none of them seems working and having an index.yaml file in them, can you send the correct URL that has index.yaml in it and is working

And the error that I am facing is 
 ` URL:       oci://8gears.container-registry.com/library/n8n
    Message:               building artifact
    Observed Generation:   1
    Reason:                ProgressingWithRetry
    Status:                True
    Type:                  Reconciling
    Last Transition Time:  2024-11-29T12:26:15Z
    Message:               failed to fetch Helm repository index: failed to cache index to temporary file: object required
    Observed Generation:   1
    Reason:                Failed
    Status:                False
    Type:                  Ready
    Last Transition Time:  2024-11-29T12:23:05Z
    Message:               failed to fetch Helm repository index: failed to cache index to temporary file: object required
    Observed Generation:   1
    Reason:                Failed
    Status:                True
    Type:                  FetchFailed
  Observed Generation:     -1
Events:
  Type     Reason  Age                  From               Message
  ----     ------  ----                 ----               -------
  Warning  Failed  85s (x8 over 4m35s)  source-controller  failed to fetch Helm repository index: failed to cache index to temporary file: object required`

**Comment by Vad1mo** [comment]
This chart has only supports OCI, so there is no index.yaml. See how to [use OCI-based registries
](https://helm.sh/docs/topics/registries/)

This should work
```sh
helm install my-n8n oci://8gears.container-registry.com/library/n8n --version 0.25.2
```
